packages/grid/_modules_/grid/components/toolbar/GridFilterToolbarButton.tsx

```diff
@@ -11,9 +13,21 @@ import {
 import { gridPreferencePanelStateSelector } from '../../hooks/features/preferencesPanel/gridPreferencePanelSelector';
 import { GridPreferencePanelsValue } from '../../hooks/features/preferencesPanel/gridPreferencePanelsValue';
 import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { GridTranslationKeys } from '../../models/api/gridLocaleTextApi';
 import { GridApiContext } from '../GridApiContext';
 
+const useStyles = makeStyles(
+  (theme) => ({
+    list: {
+      margin: theme.spacing(1),
+      padding: theme.spacing(0, 1),
+    },
+  }),
+  { name: 'MuiDataGridFilterButtonTooltipList' },
+);
+
 export const GridFilterToolbarButton: React.FC<{}> = () => {
```

Comment:
might as well fix react FC?!

Reply:
I have a separate ticket for that (@oliviertassinari has created one). I'll handle it separately as updates in more components are needed.

Reply:
I know there is a separate issue for that, however, it might not be a priority ATM.
My point is to leave the code slightly cleaner than when you found it.
So if there is a small thing like this in a file you refactor, I think we should clean it. Same for the naming conventions... Don't need to wait for the issue, if the file is not named according to the convention then rename it. No need to wait for an issue...
If we all do like this, then the separate issue might get solved just by cleaning things bits by bits on the go.

Reply:
+1 for the Scout approach as general principle: leaving things in a better shape than when we found them.

On this line, #1369 and #1325 are relevant. I guess it depends on where we want to stop. The visual aspect was already at the border of the problem we aim to fix.

Reply:
Updated